modular_skyrat/modules/cme/code/cme.dm

```diff
@@ -121,6 +121,8 @@
 	else
 		switch(cme_intensity)
 			if(CME_UNKNOWN)
+				if(SSsecurity_level.get_current_level_as_number() < SEC_LEVEL_RED)
```

Comment:
atomize these changes out, they have nothing to do with the PR

Reply:
Okay